pages/_app.js

```diff
@@ -22,9 +22,15 @@ import withData from '../lib/withData';
 
 import theme from '../lib/theme';
 
-import 'bootstrap/dist/css/bootstrap.min.css'; // eslint-disable-line node/no-unpublished-import
-import 'nprogress/nprogress.css'; // eslint-disable-line node/no-unpublished-import
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'nprogress/nprogress.css';
+import 'react-datetime/css/react-datetime.css';
```

Comment:
The classes are all specific to the module such `.rdt`, `.rdtPicker` so it's fine if we avoid such classes.